src/components/application_manager/test/resumption/resume_ctrl_test.cc

```diff
@@ -708,13 +711,13 @@ TEST_F(ResumeCtrlTest, SaveAllApplications) {
   DataAccessor<application_manager::ApplicationSet> accessor(app_set,
                                                              app_set_lock_);
 
-  EXPECT_CALL(app_mngr_, applications())
-      .WillRepeatedly(Return(accessor));
-  EXPECT_CALL(*mock_storage, SaveApplication(Eq(test_app)));
-  res_ctrl->SaveAllApplications();
+  ON_CALL(app_mngr_, applications())
+      .WillByDefault(Return(accessor));
+  EXPECT_CALL(*mock_storage_, SaveApplication(Eq(test_app)));
+  res_ctrl_->SaveAllApplications();
 
   // Fix DataAccessor release and WinQt crash
```

Comment:
@LitvinenkoIra This comment should be deleted too, since it was related to VerifyAndClearExpectations()